### ISSUE TITLE
Are you done with qpp conversion tool 2023 updates?

### ISSUE THREAD

**Issue by abhisheka1993** [CMSgov/qpp-conversion-tool#1379]
Are you done with qpp conversion tool 2023 updates?
If not then when you will be finished it with?

_Originally posted by @abhisheka1993 in https://github.com/CMSgov/qpp-conversion-tool/issues/1364#issuecomment-1787057931_

**Comment by abhisheka1993** [comment]
did you get a chance to check it?

**Comment by saquino0827** [comment]
Hi @abhisheka1993,
We have majority of the work complete but are waiting on a requirements update for PCF program submissions. I estimate this will be finished within the next week. 2023 MIPS should be completed. Feel free to open back up another issue if anything else comes up.

**Comment by abhisheka1993** [comment]
Thanks saquino0827, I am just waiting for the updates for 2023.
